src/libraries/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -182,7 +182,17 @@ public void GetTempFileName()
             try
             {
                 Assert.True(File.Exists(tmpFile));
-                Assert.Equal(".tmp", Path.GetExtension(tmpFile), ignoreCase: true, ignoreLineEndingDifferences: false, ignoreWhiteSpaceDifferences: false);
+                string fileName = Path.GetFileName(tmpFile);
+                Assert.StartsWith("tmp", fileName);
+                Assert.Equal("tmpXXXXXX.tmp".Length, fileName.Length);
+                Assert.EndsWith(".tmp", fileName);
+
+                const string validChars = "abcdefghijklmnopqrstuvwxyz0123456789";
+                for (int i = 3; i < 9; i++)
+                {
+                    Assert.True(validChars.Contains(char.ToLowerInvariant(fileName[i])));
```

Comment:
Why is the char.ToLowerInvariant here?  We're only using lowercase letters; there shouldn't be any uppercase in the generated name.

Reply:
unix.

Reply:
Then why not just add "ABCDEFGHIJKLMNOPQRSTUVWXYZ" to the ValidChars rather than using char.ToLowerInvariant?

Reply:
Because it has the same effect and is shorter. I can change if you like, I have no preference